.github/workflows/push-helm-chart.yaml

```diff
@@ -18,8 +18,9 @@ jobs:
   verify-and-push-helm-chart:
 
     permissions:
-      contents: read
-      packages: write # to push artifacts to `ghcr.io`
+      contents: write # needed to pull git repo and create "chart-release"
```

Comment:
can u pls elaborate why this is needed?

Reply:
I was looking at other projects that implemented oci and cosign and saw it was used. Anyways I reverted the change since I saw https://github.com/spegel-org/spegel/blob/main/.github/workflows/release.yaml uses the same permissions

Reply:
@MuneebAijaz any update?